reactApp/parent.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Child from './child.jsx'
+class Parent extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {name: "", array: []};
+
+    this.handleButton = this.handleButton.bind(this);
+  }
+  
+  handleButton(event) {
+    let array = this.state.array;
+    let myName = this.refs.myName;
+    if (!array.find((element) => (myName.value.trim() == "" || (element == myName.value.trim())))) { 
```

Comment:
@VARTIKASINGH1234 We can check this condition `myName.value.trim() == ""` before the array.find ?